builder/vmware/common/step_create_disks.go

```diff
@@ -44,9 +44,14 @@ func (s *StepCreateDisks) Run(ctx context.Context, state multistep.StateBag) mul
 		diskSizes = append(diskSizes, fmt.Sprintf("%dM", uint64(s.MainDiskSize)))
 	}
 	// Additional disks
+	incrementer := 1
 	if len(s.AdditionalDiskSize) > 0 {
 		for i, diskSize := range s.AdditionalDiskSize {
-			path := filepath.Join(*s.OutputDir, fmt.Sprintf("%s-%d.vmdk", s.DiskName, i+1))
+			// scsi slot 7 is reserved, so we skip it.
+			if i+incrementer == 7 {
+				incrementer = 2
+			}
+			path := filepath.Join(*s.OutputDir, fmt.Sprintf("%s-%d.vmdk", s.DiskName, i+incrementer))
 			diskFullPaths = append(diskFullPaths, path)
 			size := fmt.Sprintf("%dM", uint64(diskSize))
 			diskSizes = append(diskSizes, size)
```

Comment:
Is there a limit that we should document here? Googling seems to indicate that we can get up to 15 devices on the scsi controller with 0:7. Creating a vm with 15 additional disks does hang but maybe I'm using the wrong config.

Reply:
I don't know. I was able to create a vm with 15 disks successfully. Then I tried 16 and that worked too. If there's an upper bound I'm not sure what it is.

Reply:
Yeah - its wired I dont get an error it just hangs there. I'll let it sit longer to see if anything changes. But I don't think it is a blocker.